storage/database/dynamodb_test.go

```diff
@@ -85,6 +90,62 @@ func (s *SuiteDynamoDB) TestDynamoDB_Put() {
 	s.NoError(returnedErr)
 }
 
+func (s *SuiteDynamoDB) TestDynamoDB_Timeout() {
+	// fakeEndpoint allows TCP handshakes, but doesn't answer anything to client.
+	// The fake server is used to produce a network failure scenario.
+	fakeEndpoint := "127.0.0.1:14566"
```

Comment:
Could you make a variable for max tries and timeout? It would help a lot figuring out this test.
```suggestion
	fakeEndpoint := "127.0.0.1:14566"
	maxRetries := 2
	timeout = 1 * time.Second
```
```
			if err := listen.SetDeadline(time.Now().Add((maxRetries+1) * timeout * 2)); err != nil {
```

Reply:
Applied.